docs/source/index.rst

```diff
@@ -0,0 +1,43 @@
+.. qibo-tii-provider documentation master file, created by
+   sphinx-quickstart on Fri Dec 15 11:46:09 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to qibo-tii-provider's documentation!
+=============================================
+
+Qqibo-tii-provider is the front end interface to the Qibo labs.
+
+The main purpose of the project is to create a client tool written in python
+able to launch quantum computations through HTTP.
+
+Example usage
```

Comment:
We can move this to a section just after installation instructions, e.g. tutorials.